extensions/amp-story/0.1/amp-story.js

```diff
@@ -717,14 +717,15 @@ export class AmpStory extends AMP.BaseElement {
    * @return {?string} The URL of the background resource
    */
   getBackgroundUrl_(pageElement) {
-    const fillElement =
-        scopedQuerySelector(pageElement, '[template="fill"]');
+    let fillElement;
 
-    if (!fillElement) {
-      return;
+    try {
+      fillElement = dev().assertElement(
+          scopedQuerySelector(pageElement, '[template="fill"]'));
+    } catch (e) {
+      return '';
```

Comment:
Why not just remove the `dev().assertElement`?  By default, `scopedQuerySelector` should return `null` if the element wasn't found.

Reply:
its more or less about annotations, I tried that but `scopedQuerySelector` expects an element but closure complains complains that the passed argument might be null

Reply:
Then maybe we can do:

```javascript
  const fillElementOrNull = scopedQuerySelector(pageElement, '[template="fill"]');

  if (!fillElement) {
    return null;
  }

  const fillElement = dev().assertElement(fillElementOrNull);
  const fillPosterElement = scopedQuerySelector(fillElement, '[poster]');
  const srcElement = scopedQuerySelector(fillElement, '[src]');
```

Or, just cast it:


```javascript
  const fillElement = scopedQuerySelector(pageElement, '[template="fill"]');

  if (!fillElement) {
    return null;
  }

  const fillPosterElement = scopedQuerySelector(/** @type {!Element} */ (fillElement), '[poster]');
  const srcElement = scopedQuerySelector(/** @type {!Element} */ (fillElement), '[src]');
```

Reply:
Done